reporter/html.js

```diff
@@ -782,9 +782,11 @@ export function escapeText( s ) {
 					escapeText( actual ) + "</pre></td></tr>";
 
 				if ( typeof details.actual === "number" && typeof details.expected === "number" ) {
-					showDiff = true;
-					diff = ( details.actual - details.expected );
-					diff = ( diff > 0 ? "+" : "" ) + diff;
+					if ( !isNaN( details.actual ) && !isNaN( details.expected ) ) {
```

Comment:
@leobalter I actually like this PR's original output for NaN cases (i.e., before introducing the isNaN checks):
> **Expected:** `NaN`
> **Result:** `42`
> **Diff**: `NaN`